domain/upgrade/state/state_test.go

```diff
@@ -326,3 +326,13 @@ func (s *stateSuite) TestActiveUpgradesSingular(c *gc.C) {
 	c.Assert(err, jc.ErrorIsNil)
 	c.Check(activeUpgrade, gc.Equals, uuid)
 }
+
+func (s *stateSuite) TestSetDBUpgradeCompleted(c *gc.C) {
+	uuid, err := s.st.CreateUpgrade(context.Background(), version.MustParse("3.0.0"), version.MustParse("3.0.1"))
+	c.Assert(err, jc.ErrorIsNil)
+	err = s.st.SetDBUpgradeCompleted(context.Background(), uuid)
+	c.Assert(err, jc.ErrorIsNil)
+
```

Comment:
You could call it again and assert that you get a `NotFound` error, to test the negative case.

Reply:
done